math/extended_euclid_algorithm.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+// Finding coefficients of a and b ie x and y in gcd(a, b) = a * x + b * y
+// d is gcd(a, b)
+// This is also used in finding Modular multiplicative inverse of a number
+// (A * B)%M == 1 Here B is the MMI of A for given M, so extendedEuclid (A, M) gives B
```

Comment:
math/extended_euclid_algorithm.cpp:5:  Lines should be <= 80 characters long  [whitespace/line_length] [2]